amethyst_input/src/axis.rs

```diff
@@ -1,6 +1,9 @@
 use serde::{Deserialize, Serialize};
 
-use super::{Button, ControllerAxis};
+use super::{
+    mouse::{MouseAxis, MouseMode},
+    Button, ControllerAxis,
+};
 
 /// Represents any input represented by a float value from -1 to 1.
```

Comment:
If we're going to introduce axes that break this rule, then this documentation needs to be updated.